mobile/src/feeds/home-feeds-screen.tsx

```diff
@@ -72,7 +74,7 @@ class HomeFeedsScreenImpl extends React.Component<FeedsScreenProps & FeedScreenD
     }
 
     public render() {
-        const feeds = this.sortedFeeds().filter(this.feedPredicate);
+        const feeds = this.sortedFeeds().filter((feed) => HomeFeedsScreenImpl.feedPredicate(feed, this.props.filter));
```

Comment:
Check for working the last commit. Probably feedPredicate will be undefined due to babel